README.md

```diff
@@ -1,11 +1,11 @@
-# Search API Pantheon: Solr 8 & Drupal 9/10 Integration
+# Search API Pantheon: Solr 8 & Drupal 10/11 Integration
 
 [![Search API Pantheon](https://github.com/pantheon-systems/search_api_pantheon/actions/workflows/ci.yml/badge.svg?branch=8.x)](https://github.com/pantheon-systems/search_api_pantheon/actions/workflows/ci.yml)
 [![Limited Availability](https://img.shields.io/badge/Pantheon-Limited_Availability-yellow?logo=pantheon&color=FFDC28)](https://pantheon.io/docs/oss-support-levels#limited-availability)
 
 ## Important Notice - Schema Reversion Prevention
 
-Starting with version 8.2, this module includes critical fixes to prevent Solr schema reversions that could cause:
+Starting with version 8.3, this module includes critical fixes to prevent Solr schema reversions that could cause:
```

Comment:
This started with 8.2 shouldn't need to change.
```suggestion
Starting with version 8.2, this module includes critical fixes to prevent Solr schema reversions that could cause:
```